app/src/main/java/com/github/sdpteam15/polyevents/helper/GoogleMapHelper.kt

```diff
@@ -0,0 +1,429 @@
+package com.github.sdpteam15.polyevents.helper
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.drawable.Drawable
+import android.util.Log
+import androidx.core.content.ContextCompat
+import com.github.sdpteam15.polyevents.R
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.*
+import kotlin.math.pow
+
+enum class PolygonAction {
+    RIGHT,
+    DOWN,
+    DIAG,
+    MOVE,
+    ROTATE
+}
+
+object GoogleMapHelper {
+    var context: Context? = null
+    var map: GoogleMap? = null
+    var uid = 1
+
+    //Attributes that can change
+    var minZoom = 17f
+    var maxZoom = 21f
+
+    var swBound = LatLng(46.519941764550545, 6.564997248351575)  // SW bounds
+    var neBound = LatLng(46.5213428130699, 6.566603220999241)    // NE bounds
```

Comment:
Ok for now, but maybe we could let the admin define the boundaries ? (This will be for a later task)